shared/store/wallet.ts

```diff
@@ -162,3 +167,12 @@ export const tokenSelector = createSelector([tokensSelector, selectTokenId], (to
     return token.id === tokenId
   })
 })
+
+/**
+ * Get single token detail by `displaySymbol` from wallet store.
```

Comment:
What's the purpose of having a selector here?

Reply:
here selector will provide us token details on the basis of `displaySymbol` we can remove it but then we have to filter required token details from all tokens in the `_Default` icon component

Reply:
I'm a bit hesitant about this approach. This might cause performance issue especially looping on the list of allTokens every time. Can we just pass the required fields directly? Since most likely they won't change